packages/web-react/src/components/Collapse/__tests__/useResizeHeight.test.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+Postponed because we are unable to simulate and subsequently test the correct behavior of the useResizeObserver hook.
```

Comment:
So… do we test anything here? 🤔 This suggests no tests in this file work. But apparently, the code below doesn't seem to be doing nothing…

I'd propose either to reword the comment and move it to the lines we are struggling with, including a reference to the issue to tackle it later, or delete the test entirely.

Reply:
There is a linked issue to this case. We do not want to throw away that code because we tried hard to make it happen. But it needs more time. So we created the https://jira.lmc.cz/browse/DS-549, where we should make this work or replace this with a visual test.

Yet commented entirely and documented.